src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/CustomHostTests.cs

```diff
@@ -36,7 +36,7 @@ public void FilePathIsPopulatedCorrectly()
         private static void MakeAssemblyGetEntryAssemblyReturnNull()
         {
             typeof(Assembly)
```

Comment:
This should call `Assembly.SetEntryAssembly(null)` instead of private reflection.

Reply:
I was actually wondering that something didn't look quite right there, but couldn't put my finger on it. Will fix